src/basic/process-util.c

```diff
@@ -781,13 +781,13 @@ int pid_get_start_time(pid_t pid, uint64_t *ret) {
                 return -EIO;
 
         if (ret)
-                *ret = llu;
+                *ret = jiffies_to_usec(llu);
 
         return 0;
 }
 
-int pidref_get_start_time(const PidRef *pid, uint64_t *ret) {
-        uint64_t t;
+int pidref_get_start_time(const PidRef *pid, usec_t *ret) {
```

Comment:
if you return a proper time here, then maybe add a comment somewhere in which clock. If i read the kernel sources correctly it's CLOCK_BOOTTIME.